src/components/colorPicker/index.js

```diff
@@ -34,9 +35,10 @@ class ColorPicker extends PureComponent {
      */
     colors: PropTypes.arrayOf(PropTypes.string),
     /**
-     * The value of the selected swatch // TODO: rename prop 'selectedValue'
+     * The value of the selected swatch // TODO: remove after migration to 'selectedValue'
      */
     value: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
+    selectedValue: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
```

Comment:
What's the reason for the rename? 
personally, `value` and `selectedValue` feel the same. 
Do you want change it for readability purpose? cause that will just add another migration for us..

Reply:
It is for readability. Ok, I can leave it if it doesn't feel necessary.